web/api/js/codechecker-api-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "codechecker-api-js",
-  "version": "6.24.0",
+  "version": "6.25.0",
```

Comment:
Do not we need to update the version in the following files?
https://github.com/Ericsson/codechecker/blob/2d33728f9a0398e6724b5ddd48ca4fe041264533/web/codechecker_web/shared/version.py#L21
https://github.com/Ericsson/codechecker/blob/2d33728f9a0398e6724b5ddd48ca4fe041264533/web/server/www/scripts/version.js#L1